Resources/Prototypes/_RMC14/Roles/Loadouts/Items/RoleSpecific/synthetic.yml

```
@@ -0,0 +1,311 @@
+# Synthetic
+
+# UNMC Jumpsuits
+- type: loadout
+  id: ScrubsBlueRMC
+  storage:
+    back:
+    - CMScrubsBlue
```

```suggestion
  equipment:
    jumpsuit: CMScrubsBlue
```
Unless it is intentional, you should do this for each of the jumpsuits so that they are worn upon spawning, instead of being stored in the backpack